requirements.txt

```diff
@@ -1,21 +1,8 @@
-CacheControl==0.12.5
-certifi==2018.8.24
-chardet==3.0.4
-diskcache==3.0.6
-EsiPy==0.5.0
-idna==2.7
-lockfile==0.12.2
-msgpack==0.5.6
-numpy==1.15.1
-pandas==0.23.4
-pyaml==17.12.1
-pyswagger==0.8.39
-python-dateutil==2.7.3
-pytz==2018.5
-PyYAML==3.13
-requests==2.19.1
-six==1.11.0
-slackclient==1.2.1
-urllib3==1.23
-validate-email==1.3
-websocket-client==0.53.0
+esipy
```

Comment:
requirements.txt is _generally_ used for pinning versions of the entire environment. Doesn't matter too much in this version, so I've kept it, but also added a setup.py for easier setup on the server.